docs/fsharp/using-fsharp-on-azure/table-storage.md

```diff
@@ -41,9 +54,13 @@ Add the following `open` statements to the top of the `tables.fsx` file:
 
 [!code-fsharp[TableStorage](../../../samples/snippets/fsharp/azure/table-storage.fsx#L1-L5)]
 
-### Get your connection string
+### Get your Azure Storage connection string
+
+If you are connecting to Azure Storage Table service, you'll need your connection string for this tutorial. You can copy your connection string from the Azure portal. For more information about connection strings, see [Configure Storage Connection Strings](/azure/storage/storage-configure-connection-string).
 
-You'll need an Azure Storage connection string for this tutorial. For more information about connection strings, see [Configure Storage Connection Strings](/azure/storage/storage-configure-connection-string).
+### Get your Azure Cosmos DB connection string
+
+If you are connecting to Azure Cosmos DB, you'll need your connection string for this tutorial. You can copy your connection string from the Azure portal. In the Azure portal, in your Cosmos DB account, go to **Settings** > **Connection String**, and click the **Copy** button to copy your Primary Connection String. 
 
 For the tutorial, you'll enter your connection string in your script, like this:
 
```

Comment:
Some existing issues pre-acrolinx:

line 82 : This will return -> This returns

line 84: will be stored -> are stored

line 92: Now we'll add our `Customer` to the table. To do so, you create a `TableOperation` that will execute on the table. ->
Now add `Customer` to the table. To do so, create a `TableOperation` that executes on the table.